COPYRIGHT

```diff
@@ -81,6 +86,7 @@ Copyright (C) 1988-2018 by its authors, which include:
 * Laurent Bartholdi
 * Lucas Jones
 * Lukas Maas
+* Lucas Wollenhaupt
```

Comment:
Same here, c comes before k so like Lucas Jones I should be above Lukas Maas. Of course I don't really mind where I stand here and I don't want to suggest anything here just for consistency.